### ISSUE TITLE
Doesn't work with Zed

### ISSUE THREAD

**Issue by DidacGit** [xremap/xremap#523]
Hey, first of all, amazing software!

It seems like xremap can't retrieve the name of the [Zed](https://zed.dev/) window. For example, using this simple config:

```yml
keymap:
  - name: Emacs inspired
    application:
      only: ["dev.zed.Zed"]
    remap:
      C-f: { with_mark: right }
```
When I run it with xremap, this is the output, of typing `C-f` in Mousepad and then typing it on Zed:
```
application-client: X11 (supported: true)
application: mousepad.Mousepad
Reconnecting to X11 due to error: X11 error X11Error { error_kind: Window, error_code: 3, sequence: 20, bad_value: 0, minor_opcode: 0, major_opcode: 20, extension_name: None, request_name: Some("GetProperty") }
Reconnecting to X11 due to error: X11 error X11Error { error_kind: Window, error_code: 3, sequence: 2, bad_value: 0, minor_opcode: 0, major_opcode: 15, extension_name: None, request_name: Some("QueryTree") }
```
I got the `dev.zed.Zed` name by using `xprop`, by the way.
Any idea of what could be wrong? I'm on Debian 12, XFCE (X11). I have Zed installed locally, normally, so no Flatpak nor anything weird.
Thanks in advance!

**Comment by wistoft** [comment]
I could confirm the problem as you described it, and I've made a PR with a fix. It would be nice, if you could try it out, and see if it fixes your problem.

**Comment by k0kubun** [comment]
Released the patch as v0.10.1. Thank you!

**Comment by DidacGit** [comment]
Thank you guys! Now it works perfectly.
